.eslintrc

```diff
@@ -1,3 +1,5 @@
+extends: eslint:recommended
```

Comment:
I would be slightly in favor of keeping our options. `eslint:recommended` is a great baseline, but I wouldn't want our explicit choices to come to depend on the choices of the ESLint team in the future.